pkg/controller/acicontainersoperator.go

```diff
@@ -465,10 +533,45 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 					Update(context.TODO(), acicnioperator, metav1.UpdateOptions{})
 				if er != nil {
 					log.Error(er)
+					acicnioperatorsuccess = false
 				}
 			}
 		}
 	}
+	if acicnioperatorsuccess == true {
+		log.Info("Checking if accprovisioninput CR already present")
+		accprovisioninput, err := c.GetAccProvisionInputCR()
+		if err != nil {
+			log.Info("Not Present ..Creating accprovisioninput CR")
+			er := c.CreateAccProvisionInputCR()
+			if er != nil {
+				log.Error(er)
```

Comment:
Here and below, wherever we are logging error, is it okay to proceed in spite of error?

Reply:
The operator repeatedly tries to create the CR even if there's an error so we can proceed here. @abhis2112 does that sound right?